config/webhook/manifests.yaml

```diff
@@ -24,8 +26,11 @@ webhooks:
     - UPDATE
     resources:
     - namespaces
-- clientConfig:
-    caBundle: Cg==
```

Comment:
Looks like `caBundle` is removed here but they are persisted in manifest_staging.

Reply:
I think `config/overlays/mutation_webhook/webhook.yaml` is temporary location, and it's not generated. Once it's generated, it'll go to `config/webhook/manifests.yaml`

Reply:
ah ok. should we manually update this for now?

Reply:
I don't think it matters but, I updated to be same as it would have been generated.

Reply:
Related bug: #817